pytket/extensions/qiskit/qiskit_convert.py

```diff
@@ -550,15 +579,12 @@ def qiskit_to_tk(qcirc: QuantumCircuit, preserve_param_uuid: bool = False) -> Ci
     Converts a qiskit :py:class:`qiskit.QuantumCircuit` to a pytket :py:class:`Circuit`.
 
     :param qcirc: A circuit to be converted
-    :type qcirc: QuantumCircuit
```

Comment:
These type declarations in the docstrings are redundant as the types are inferred from the function/class signiture. This inference is done by the [sphinx-autodoc-typehints package](https://github.com/tox-dev/sphinx-autodoc-typehints) package which is used in the docs build.